README.md

```diff
@@ -70,6 +70,63 @@ For running integration tests in Anvil node, run `make anvil-tests`. This runs t
 
 `make anvil` (if using foundry stack). To get a list of pre-funded addresses, you can look at the beginning of the logs in the `anvil` Docker container, or reference <https://github.com/foundry-rs/foundry/blob/0d8302880b79fa9c3c4aa52ab446583dece19a34/crates/anvil/README.md?plain=1#L48>.
 
+### (NEW) Deploy & Verify
```

Comment:
Let's move the deployment and verification instructions to a separate README to keep the main README concise. In the main README, we can include a reference to the deployment README.